src/polyfem/assembler/MooneyRivlin3ParamElasticity.hpp

```diff
@@ -36,13 +36,13 @@ namespace polyfem::assembler
 
 			const T J = polyfem::utils::determinant(def_grad);
 			const T log_J = log(J);
+			const auto F_tilde = def_grad;
```

Comment:
Need to wait the other PR for these changes

Reply:
agreed